library/ssl_tls.c

```diff
@@ -6567,16 +6568,9 @@ int mbedtls_ssl_handle_message_type( mbedtls_ssl_context *ssl )
 
 int mbedtls_ssl_send_fatal_handshake_failure( mbedtls_ssl_context *ssl )
 {
-    int ret;
-
-    if( ( ret = mbedtls_ssl_send_alert_message( ssl,
-                    MBEDTLS_SSL_ALERT_LEVEL_FATAL,
-                    MBEDTLS_SSL_ALERT_MSG_HANDSHAKE_FAILURE ) ) != 0 )
-    {
-        return( ret );
-    }
-
-    return( 0 );
+    return mbedtls_ssl_send_alert_message( ssl,
+               MBEDTLS_SSL_ALERT_LEVEL_FATAL,
+               MBEDTLS_SSL_ALERT_MSG_HANDSHAKE_FAILURE );
```

Comment:
Style:
```
    return( mbedtls_ssl_send_alert_message( ssl,
￼                 MBEDTLS_SSL_ALERT_LEVEL_FATAL,
￼                 MBEDTLS_SSL_ALERT_MSG_HANDSHAKE_FAILURE ) );
```

Reply:
Thanks for reviewing.
Function arguments were re-aligned.
Change log had no lines for the next version, so I had to create 2.19.2 header for the suggested bugfix enties.

Reply:
Could you please add parentheses around the return value? i.e. `return( mbedtls_xxx( ) );`